### ISSUE TITLE
Difficult to get normalized coordinates for spatial analytics parameters

### ISSUE THREAD

**Issue by antoniomtz** [dlstreamer/pipeline-server#87]
Hello,

What would be the best way (tools) to get the normalized coordinates [0,1] in a video for the "line" value for object_line_crossing parameter?

**Comment by whbruce** [comment]
Use the [normalized_rect](https://github.com/openvinotoolkit/dlstreamer_gst/blob/master/python/gstgva/region_of_interest.py#L43) method of the DL Streamer [RegionOfInterest](https://github.com/openvinotoolkit/dlstreamer_gst/blob/master/python/gstgva/region_of_interest.py#L32) class. See example usage in [object_line_crossing.py#L50](https://github.com/intel/video-analytics-serving/blob/master/extensions/spatial_analytics/object_line_crossing.py#L50)

**Comment by antoniomtz** [comment]
@whbruce What I need to do is to identify coordinates in different areas from a videos to configure a new line in the parameter file. I want to avoid a manual process of try/error. 
I'm wondering if there is a tool to get the normalized coordinates. For example, this tool https://stackoverflow.com/questions/55149171/how-to-get-roi-bounding-box-coordinates-with-mouse-clicks-instead-of-guess-che/55153612#55153612 allows me to get the coordinates but they are not normalized as you need them for the object_line_crossing.

In other words, for the https://github.com/intel-iot-devkit/sample-videos/raw/master/people-detection.mp4 , how did you guys calculate the exact coordinates to apply the different hallways at the right place?

**Comment by whbruce** [comment]
That's a nice tool! I [tweaked it slightly](https://gist.github.com/whbruce/dcdeaa969cc8fa12dd803499074bead3) and it seemed to do the job. 

@darrindu let's find a way of updating the docs to help people do this.

**Comment by antoniomtz** [comment]
@whbruce Thank! it seems to give me the right normalized coordinates!
